### ISSUE TITLE
arm64: Unhandled user fault: addr=130 ip=fc23a9fd23dc access=r-- err=bad address

### ISSUE THREAD

**Issue by avagin** [google/gvisor#2064]
Many arm64 travis jobs fail with this error:
https://travis-ci.org/google/gvisor/jobs/658335722
```
D0304 18:14:51.139579    3598 task_run.go:309] [   1] Unhandled user fault: addr=130 ip=fc23a9fd23dc access=r-- err=bad address
2498D0304 18:14:51.139778    3598 task_log.go:81] [   1] Registers:
2499D0304 18:14:51.139892    3598 task_log.go:88] [   1] Pc       = 0000fc23a9fd23dc
2500D0304 18:14:51.139944    3598 task_log.go:88] [   1] Pstate   = 0000000000000000
2501D0304 18:14:51.139971    3598 task_log.go:88] [   1] R0       = 0000000000000130
2502D0304 18:14:51.139994    3598 task_log.go:88] [   1] R1       = 0000000000000000
2503D0304 18:14:51.140018    3598 task_log.go:88] [   1] R10      = 0101010101010101
2504D0304 18:14:51.140044    3598 task_log.go:88] [   1] R11      = 0101010101010101
2505D0304 18:14:51.140072    3598 task_log.go:88] [   1] R12      = 0000000000000015
2506D0304 18:14:51.140101    3598 task_log.go:88] [   1] R13      = ffffffffffffffff
2507D0304 18:14:51.140127    3598 task_log.go:88] [   1] R14      = ff00000000000000
2508D0304 18:14:51.140155    3598 task_log.go:88] [   1] R15      = ffffffffffffffff
2509D0304 18:14:51.140187    3598 task_log.go:88] [   1] R16      = 0000fc23aa0a1008
2510D0304 18:14:51.140228    3598 task_log.go:88] [   1] R17      = 0000fc23a9fd23b0
2511D0304 18:14:51.140278    3598 task_log.go:88] [   1] R18      = 0000000000000a03
2512D0304 18:14:51.140405    3598 task_log.go:88] [   1] R19      = 0000f6d14ddc7681
2513D0304 18:14:51.140450    3598 task_log.go:88] [   1] R2       = 0000000000000000
2514D0304 18:14:51.140482    3598 task_log.go:88] [   1] R20      = 0000000000000228
2515D0304 18:14:51.140515    3598 task_log.go:88] [   1] R21      = 0000fc23aa076658
2516D0304 18:14:51.140554    3598 task_log.go:88] [   1] R22      = 0000000000000001
2517D0304 18:14:51.140593    3598 task_log.go:88] [   1] R23      = 0000fc23aa0a29a0
2518D0304 18:14:51.140638    3598 task_log.go:88] [   1] R24      = 0000000000000000
2519D0304 18:14:51.140680    3598 task_log.go:88] [   1] R25      = 00000000ffffffff
2520D0304 18:14:51.140718    3598 task_log.go:88] [   1] R26      = 000000000000000c
2521D0304 18:14:51.140754    3598 task_log.go:88] [   1] R27      = 0000fc23aa0a22b8
2522D0304 18:14:51.140792    3598 task_log.go:88] [   1] R28      = 0000000000000016
2523D0304 18:14:51.140834    3598 task_log.go:88] [   1] R29      = 0000f6d14ddc7610
2524D0304 18:14:51.140877    3598 task_log.go:88] [   1] R3       = 0000000000000010
2525D0304 18:14:51.140912    3598 task_log.go:88] [   1] R30      = 0000fc23a9fbf2bc
2526D0304 18:14:51.140947    3598 task_log.go:88] [   1] R4       = 2e656c61636f6c2f
2527D0304 18:14:51.140989    3598 task_log.go:88] [   1] R5       = 8080808080808000
2528D0304 18:14:51.141028    3598 task_log.go:88] [   1] R6       = 0000f6d14ddc7681
2529D0304 18:14:51.141070    3598 task_log.go:88] [   1] R7       = 726168732f727375
2530D0304 18:14:51.141116    3598 task_log.go:88] [   1] R8       = 656c61636f6c2f65
2531D0304 18:14:51.141243    3598 task_log.go:88] [   1] R9       = 000000000042d010
2532D0304 18:14:51.141295    3598 task_log.go:88] [   1] Sp       = 0000f6d14ddc7610
2533D0304 18:14:51.141326    3598 task_log.go:105] [   1] Stack:
2534D0304 18:14:51.141366    3598 task_log.go:122] [   1] f6d14ddc7610: 40 76 dc 4d d1 f6 00 00 bc f2 fb a9 23 fc 00 00
2535D0304 18:14:51.141431    3598 task_log.go:122] [   1] f6d14ddc7620: 81 76 dc 4d d1 f6 00 00 70 76 dc 4d d1 f6 00 00
2536D0304 18:14:51.141484    3598 task_log.go:122] [   1] f6d14ddc7630: 24 77 dc 4d d1 f6 00 00 37 79 dc 4d d1 f6 00 00
2537D0304 18:14:51.141519    3598 task_log.go:122] [   1] f6d14ddc7640: a0 76 dc 4d d1 f6 00 00 34 e2 f8 a9 23 fc 00 00
2538D0304 18:14:51.141571    3598 task_log.go:122] [   1] f6d14ddc7650: 81 76 dc 4d d1 f6 00 00 11 00 00 00 00 00 00 00
2539D0304 18:14:51.141634    3598 task_log.go:122] [   1] f6d14ddc7660: 00 20 0a aa 23 fc 00 00 3a 00 00 00 00 00 00 00
2540D0304 18:14:51.141688    3598 task_log.go:122] [   1] f6d14ddc7670: 2f 75 73 72 2f 73 68 61 72 65 2f 6c 6f 63 61 6c
2541D0304 18:14:51.141737    3598 task_log.go:122] [   1] f6d14ddc7680: 65 2f 6c 6f 63 61 6c 65 2e 61 6c 69 61 73 00 00
2542D0304 18:14:51.141771    3598 task_log.go:122] [   1] f6d14ddc7690: 28 79 dc 4d d1 f6 00 00 f0 77 dc 4d d1 f6 00 00
2543D0304 18:14:51.141801    3598 task_log.go:122] [   1] f6d14ddc76a0: d0 78 dc 4d d1 f6 00 00 6c e7 f8 a9 23 fc 00 00
2544D0304 18:14:51.141838    3598 task_log.go:122] [   1] f6d14ddc76b0: 48 fd 09 aa 23 fc 00 00 52 8b dc 4d d1 f6 00 00
2545D0304 18:14:51.141866    3598 task_log.go:122] [   1] f6d14ddc76c0: 00 20 0a aa 23 fc 00 00 3a 00 00 00 00 00 00 00
2546D0304 18:14:51.141897    3598 task_log.go:122] [   1] f6d14ddc76d0: a0 29 0a aa 23 fc 00 00 00 00 00 00 00 00 00 00
2547D0304 18:14:51.141928    3598 task_log.go:122] [   1] f6d14ddc76e0: ff ff ff ff 00 00 00 00 0c 00 00 00 00 00 00 00
2548D0304 18:14:51.141955    3598 task_log.go:122] [   1] f6d14ddc76f0: b8 22 0a aa 23 fc 00 00 16 00 00 00 00 00 00 00
2549D0304 18:14:51.141985    3598 task_log.go:122] [   1] f6d14ddc7700: 59 8b dc 4d d1 f6 00 00 05 00 00 00 00 00 00 00
2550D0304 18:14:51.142015    3598 task_log.go:122] [   1] f6d14ddc7710: b8 22 0a aa 23 fc 00 00 16 00 00 00 00 00 00 00
2551D0304 18:14:51.142090    3598 task_log.go:122] [   1] f6d14ddc7720: 7c 00 00 00 77 00 00 00 6e 00 00 00 5b 00 00 00
2552D0304 18:14:51.142131    3598 task_log.go:122] [   1] f6d14ddc7730: e0 77 dc 4d d1 f6 00 00 9c 9b f8 a9 23 fc 00 00
2553D0304 18:14:51.142268    3598 task_log.go:122] [   1] f6d14ddc7740: 02 00 00 00 00 00 00 00 10 d0 42 00 00 00 00 00
2554D0304 18:14:51.142335    3598 task_log.go:122] [   1] f6d14ddc7750: c0 77 dc 4d d1 f6 00 00 c0 7a dc 4d d1 f6 00 00
2555D0304 18:14:51.142378    3598 task_log.go:122] [   1] f6d14ddc7760: 00 20 0a aa 23 fc 00 00 00 00 00 00 00 00 00 00
2556D0304 18:14:51.142435    3598 task_log.go:122] [   1] f6d14ddc7770: 59 8b dc 4d d1 f6 00 00 04 00 00 00 00 00 00 00
2557D0304 18:14:51.142487    3598 task_log.go:122] [   1] f6d14ddc7780: 01 00 00 00 00 00 00 00 16 00 00 00 00 00 00 00
2558D0304 18:14:51.142529    3598 task_log.go:122] [   1] f6d14ddc7790: 54 8b dc 4d d1 f6 00 00 59 8b dc 4d d1 f6 00 00
2559D0304 18:14:51.142578    3598 task_log.go:122] [   1] f6d14ddc77a0: 52 8b dc 4d d1 f6 00 00 07 00 00 00 00 00 00 00
2560D0304 18:14:51.142623    3598 task_log.go:122] [   1] f6d14ddc77b0: e0 77 dc 4d d1 f6 00 00 00 00 00 00 00 00 00 00
2561D0304 18:14:51.142668    3598 task_log.go:122] [   1] f6d14ddc77c0: 43 2e 75 74 66 38 00 00 07 00 00 00 00 00 00 00
2562D0304 18:14:51.142707    3598 task_log.go:122] [   1] f6d14ddc77d0: 52 8b dc 4d d1 f6 00 00 c0 7a dc 4d d1 f6 00 00
2563D0304 18:14:51.142746    3598 task_log.go:122] [   1] f6d14ddc77e0: 50 79 dc 4d d1 f6 00 00 dc 8f f8 a9 23 fc 00 00
2564D0304 18:14:51.142791    3598 task_log.go:122] [   1] f6d14ddc77f0: 52 8b dc 4d d1 f6 00 00 07 00 00 00 00 00 00 00
2565D0304 18:14:51.142834    3598 task_log.go:122] [   1] f6d14ddc7800: c0 7a dc 4d d1 f6 00 00 0c 00 00 00 00 00 00 00
2566D0304 18:14:51.143685    3598 task_log.go:122] [   1] f6d14ddc7810: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
2567D0304 18:14:51.143886    3598 task_log.go:122] [   1] f6d14ddc7820: ff ff ff ff 00 00 00 00 0c 00 00 00 00 00 00 00
2568D0304 18:14:51.144193    3598 task_log.go:122] [   1] f6d14ddc7830: b8 22 0a aa 23 fc 00 00 16 00 00 00 00 00 00 00
2569D0304 18:14:51.144359    3598 task_log.go:122] [   1] f6d14ddc7840: 90 21 10 aa 23 fc 00 00 38 00 00 00 00 00 00 00
2570D0304 18:14:51.144584    3598 task_log.go:122] [   1] f6d14ddc7850: 0c 00 00 00 d1 f6 00 00 b0 79 dc 4d d1 f6 00 00
2571D0304 18:14:51.144640    3598 task_log.go:122] [   1] f6d14ddc7860: b8 78 dc 4d d1 f6 00 00 07 00 00 00 00 00 00 00
2572D0304 18:14:51.144869    3598 task_log.go:122] [   1] f6d14ddc7870: 02 00 00 00 00 00 00 00 00 00 00 00 23 fc 00 00
2573D0304 18:14:51.144916    3598 task_log.go:122] [   1] f6d14ddc7880: 20 5b 0f aa 23 fc 00 00 a0 79 dc 4d d1 f6 00 00
2574D0304 18:14:51.145401    3598 task_log.go:122] [   1] f6d14ddc7890: e8 78 dc 4d d1 f6 00 00 e4 78 dc 4d d1 f6 00 00
2575D0304 18:14:51.145562    3598 task_log.go:122] [   1] f6d14ddc78a0: e0 0e 40 00 00 00 00 00 88 5b 0f aa 23 fc 00 00
2576D0304 18:14:51.145637    3598 task_log.go:122] [   1] f6d14ddc78b0: 10 09 f7 a9 23 fc 00 00 0c 15 40 00 00 00 00 00
2577D0304 18:14:51.147097    3598 task_log.go:122] [   1] f6d14ddc78c0: b0 79 dc 4d d1 f6 00 00 2a 76 85 01 00 00 00 00
2578D0304 18:14:51.147367    3598 task_log.go:122] [   1] f6d14ddc78d0: 50 79 dc 4d d1 f6 00 00 e8 8f f8 a9 23 fc 00 00
2579D0304 18:14:51.147527    3598 task_log.go:122] [   1] f6d14ddc78e0: 52 8b dc 4d d1 f6 00 00 07 00 00 00 00 00 00 00
2580D0304 18:14:51.147577    3598 task_log.go:122] [   1] f6d14ddc78f0: c0 7a dc 4d d1 f6 00 00 0c 00 00 00 00 00 00 00
2581D0304 18:14:51.147610    3598 task_log.go:122] [   1] f6d14ddc7900: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
2582D0304 18:14:51.147920    3598 task_log.go:122] [   1] f6d14ddc7910: ff ff ff ff 00 00 00 00 0c 00 00 00 00 00 00 00
2583D0304 18:14:51.147962    3598 task_log.go:122] [   1] f6d14ddc7920: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
2584D0304 18:14:51.148121    3598 task_log.go:122] [   1] f6d14ddc7930: 00 00 00 00 00 00 00 00 f0 b5 42 00 00 00 00 00
2585D0304 18:14:51.148501    3598 task_log.go:122] [   1] f6d14ddc7940: 98 8d 41 00 00 00 00 00 02 00 00 00 00 00 00 00
2586D0304 18:14:51.148645    3598 task_log.go:122] [   1] f6d14ddc7950: d0 79 dc 4d d1 f6 00 00 40 88 f8 a9 23 fc 00 00
2587D0304 18:14:51.148732    3598 task_log.go:122] [   1] f6d14ddc7960: 0c 00 00 00 00 00 00 00 60 00 00 00 00 00 00 00
2588D0304 18:14:51.148916    3598 task_log.go:122] [   1] f6d14ddc7970: 78 72 07 aa 23 fc 00 00 0d 00 00 00 00 00 00 00
2589D0304 18:14:51.148975    3598 task_log.go:122] [   1] f6d14ddc7980: 60 7a dc 4d d1 f6 00 00 00 60 0a aa 23 fc 00 00
2590D0304 18:14:51.149064    3598 task_log.go:122] [   1] f6d14ddc7990: ff ff ff ff 00 00 00 00 0c 00 00 00 00 00 00 00
2591D0304 18:14:51.149100    3598 task_log.go:122] [   1] f6d14ddc79a0: 52 8b dc 4d d1 f6 00 00 78 72 07 aa 23 fc 00 00
2592D0304 18:14:51.149123    3598 task_log.go:122] [   1] f6d14ddc79b0: e0 98 41 00 00 00 00 00 68 27 0a aa 23 fc 00 00
2593D0304 18:14:51.149147    3598 task_log.go:122] [   1] f6d14ddc79c0: 06 00 00 00 00 00 00 00 00 60 0a aa 23 fc 00 00
2594D0304 18:14:51.149177    3598 task_log.go:122] [   1] f6d14ddc79d0: 30 7b dc 4d d1 f6 00 00 28 2e 40 00 00 00 00 00
2595D0304 18:14:51.149207    3598 task_log.go:122] [   1] f6d14ddc79e0: 48 80 dc 4d d1 f6 00 00 01 00 00 00 00 00 00 00
2596D0304 18:14:51.149270    3598 task_log.go:122] [   1] f6d14ddc79f0: 00 b0 42 00 00 00 00 00 f0 b5 42 00 00 00 00 00
2597D0304 18:14:51.149308    3598 task_log.go:122] [   1] f6d14ddc7a00: 98 8d 41 00 00 00 00 00 02 00 00 00 00 00 00 00
2598D0304 18:14:51.149342    3598 task_log.go:65] [   1] Mappings:
2599VMAs:
260000400000-0041b000 r-xp 00000000 00:0e 44                                 /bin/ls
26010042a000-0042b000 r--p 0001a000 00:0e 44                                 /bin/ls
26020042b000-0042c000 rw-p 0001b000 00:0e 44                                 /bin/ls
26030042c000-0042d000 rw-p 00000000 00:00 0 
26040042d000-0044e000 rw-p 00000000 00:00 0                                  [heap]
2605f6d14d5c9000-f6d14ddc9000 rw-p 00000000 00:00 0                          [stack]
2606fc23a9a00000-fc23a9cd8000 r--p 00000000 00:0e 65                         /usr/lib/locale/locale-archive
2607fc23a9eb0000-fc23a9ec7000 r-xp 00000000 00:0e 62                         /lib/aarch64-linux-gnu/libpthread-2.23.so
2608fc23a9ec7000-fc23a9ed6000 ---p 00017000 00:0e 62                         /lib/aarch64-linux-gnu/libpthread-2.23.so
2609fc23a9ed6000-fc23a9ed7000 r--p 00016000 00:0e 62                         /lib/aarch64-linux-gnu/libpthread-2.23.so
2610fc23a9ed7000-fc23a9ed8000 rw-p 00017000 00:0e 62                         /lib/aarch64-linux-gnu/libpthread-2.23.so
2611fc23a9ed8000-fc23a9edc000 rw-p 00000000 00:00 0 
2612fc23a9edc000-fc23a9ede000 r-xp 00000000 00:0e 60                         /lib/aarch64-linux-gnu/libdl-2.23.so
2613fc23a9ede000-fc23a9eed000 ---p 00002000 00:0e 60                         /lib/aarch64-linux-gnu/libdl-2.23.so
2614fc23a9eed000-fc23a9eee000 r--p 00001000 00:0e 60                         /lib/aarch64-linux-gnu/libdl-2.23.so
2615fc23a9eee000-fc23a9eef000 rw-p 00002000 00:0e 60                         /lib/aarch64-linux-gnu/libdl-2.23.so
2616fc23a9eef000-fc23a9f4e000 r-xp 00000000 00:0e 58                         /lib/aarch64-linux-gnu/libpcre.so.3.13.2
2617fc23a9f4e000-fc23a9f5e000 ---p 0005f000 00:0e 58                         /lib/aarch64-linux-gnu/libpcre.so.3.13.2
2618fc23a9f5e000-fc23a9f5f000 r--p 0005f000 00:0e 58                         /lib/aarch64-linux-gnu/libpcre.so.3.13.2
2619fc23a9f5f000-fc23a9f60000 rw-p 00060000 00:0e 58                         /lib/aarch64-linux-gnu/libpcre.so.3.13.2
2620fc23a9f60000-fc23aa08e000 r-xp 00000000 00:0e 56                         /lib/aarch64-linux-gnu/libc-2.23.so
2621fc23aa08e000-fc23aa09d000 ---p 0012e000 00:0e 56                         /lib/aarch64-linux-gnu/libc-2.23.so
2622fc23aa09d000-fc23aa0a1000 r--p 0012d000 00:0e 56                         /lib/aarch64-linux-gnu/libc-2.23.so
2623fc23aa0a1000-fc23aa0a3000 rw-p 00131000 00:0e 56                         /lib/aarch64-linux-gnu/libc-2.23.so
2624fc23aa0a3000-fc23aa0a7000 rw-p 00000000 00:00 0 
2625fc23aa0a7000-fc23aa0c1000 r-xp 00000000 00:0e 54                         /lib/aarch64-linux-gnu/libselinux.so.1
2626fc23aa0c1000-fc23aa0d0000 ---p 0001a000 00:0e 54                         /lib/aarch64-linux-gnu/libselinux.so.1
2627fc23aa0d0000-fc23aa0d1000 r--p 00019000 00:0e 54                         /lib/aarch64-linux-gnu/libselinux.so.1
2628fc23aa0d1000-fc23aa0d2000 rw-p 0001a000 00:0e 54                         /lib/aarch64-linux-gnu/libselinux.so.1
2629fc23aa0d2000-fc23aa0d4000 rw-p 00000000 00:00 0 
2630fc23aa0d4000-fc23aa0f0000 r-xp 00000000 00:0e 51                         /lib/aarch64-linux-gnu/ld-2.23.so
2631fc23aa0f2000-fc23aa0f7000 rw-p 00000000 00:00 0 
2632fc23aa0fd000-fc23aa0fe000 r--p 00000000 00:00 0                          [vvar]
2633fc23aa0fe000-fc23aa100000 r-xp 00000000 00:00 0 
2634fc23aa100000-fc23aa101000 r--p 0001c000 00:0e 51                         /lib/aarch64-linux-gnu/ld-2.23.so
2635fc23aa101000-fc23aa103000 rw-p 0001d000 00:0e 51                         /lib/aarch64-linux-gnu/ld-2.23.so
2636PMAs:
263700400000-0041a000 r-xs 00000000 *fsutil.CachingInodeOperations
26380041a000-0041b000 r-xp 00003000 *pgalloc.MemoryFile
26390042a000-0042b000 r--p 00004000 *pgalloc.MemoryFile
26400042b000-0042c000 rw-p 00005000 *pgalloc.MemoryFile
26410042c000-0042d000 rw-p 00211000 *pgalloc.MemoryFile
26420042d000-0044e000 rw-p 001f0000 *pgalloc.MemoryFile
2643f6d14dc00000-f6d14ddc9000 rw-p 0000a000 *pgalloc.MemoryFile
2644fc23a9a00000-fc23a9cd8000 r--s 00000000 *fsutil.CachingInodeOperations
2645fc23a9eb0000-fc23a9ec7000 r-xs 00000000 *fsutil.CachingInodeOperations
2646fc23a9ed6000-fc23a9ed7000 r--p 001e7000 *pgalloc.MemoryFile
2647fc23a9ed7000-fc23a9edc000 rw-p 001e8000 *pgalloc.MemoryFile
2648fc23a9edc000-fc23a9ede000 r-xs 00000000 *fsutil.CachingInodeOperations
2649fc23a9eed000-fc23a9eee000 r--p 001e5000 *pgalloc.MemoryFile
2650fc23a9eee000-fc23a9eef000 rw-p 001e6000 *pgalloc.MemoryFile
2651fc23a9eef000-fc23a9f4e000 r-xs 00000000 *fsutil.CachingInodeOperations
2652fc23a9f5e000-fc23a9f5f000 r--p 001e2000 *pgalloc.MemoryFile
2653fc23a9f5f000-fc23a9f60000 rw-p 001e3000 *pgalloc.MemoryFile
2654fc23a9f60000-fc23aa08e000 r-xs 00000000 *fsutil.CachingInodeOperations
2655fc23aa09d000-fc23aa0a1000 r--p 001d8000 *pgalloc.MemoryFile
2656fc23aa0a1000-fc23aa0a7000 rw-p 001dc000 *pgalloc.MemoryFile
2657fc23aa0a7000-fc23aa0c1000 r-xs 00000000 *fsutil.CachingInodeOperations
2658fc23aa0d0000-fc23aa0d1000 r--p 001d4000 *pgalloc.MemoryFile
2659fc23aa0d1000-fc23aa0d4000 rw-p 001d5000 *pgalloc.MemoryFile
2660fc23aa0d4000-fc23aa0ef000 r-xs 00000000 *fsutil.CachingInodeOperations
2661fc23aa0ef000-fc23aa0f0000 r-xp 00006000 *pgalloc.MemoryFile
2662fc23aa0f2000-fc23aa0f3000 rw-p 001ef000 *pgalloc.MemoryFile
2663fc23aa0f3000-fc23aa0f4000 rw-p 001ee000 *pgalloc.MemoryFile
2664fc23aa0f4000-fc23aa0f5000 rw-p 001ed000 *pgalloc.MemoryFile
2665fc23aa0f5000-fc23aa0f6000 rw-p 001e4000 *pgalloc.MemoryFile
2666fc23aa0f6000-fc23aa0f7000 rw-p 001d3000 *pgalloc.MemoryFile
2667fc23aa0fe000-fc23aa100000 r-xs 00000000 *pgalloc.MemoryFile
2668fc23aa100000-fc23aa101000 r--p 00007000 *pgalloc.MemoryFile
2669fc23aa101000-fc23aa103000 rw-p 00008000 *pgalloc.MemoryFile
2670
2671D0304 18:14:51.150464    3598 task_log.go:67] [   1] FDTable:
2672	fd:0 => name host:[1]
2673	fd:1 => name host:[2]
2674	fd:2 => name host:[3]
2675
2676
```

@lubinszARM @xiaobo55x can you help to investigate what is wrong here

**Comment by xiaobo55x** [comment]
@avagin, how to reproduce the issue? by the following command? I tried to run a ls command on my machine and it works well.
```
docker run --runtime=runsc --rm -it ubuntu ls
```

**Comment by xiaobo55x** [comment]
Actually, the gVisor can run a ubuntu docker image on arm64 successfully now!
```
root@entos-qualcomm-01:/home/xuhaibo# docker run --runtime=runsc --rm -it ubuntu
root@59f4f3e77669:/# ls
bin  boot  dev  etc  home  lib  media  mnt  opt  proc  root  run  sbin  srv  sys  tmp  usr  var
root@59f4f3e77669:/# uname -a
Linux 59f4f3e77669 4.4.0 #1 SMP Sun Jan 10 15:06:54 PST 2016 aarch64 aarch64 aarch64 GNU/Linux
root@59f4f3e77669:/#
```

**Comment by xiaobo55x** [comment]
I think the cause of this issue may be related to the TLS support on arm64. Could you try to cherry-pick my TLS PR(#1832) to verify whether it can fix your issue?

**Comment by lubinszARM** [comment]
Hi @xiaobo55x @avagin 

I test it locally with the a endless-loop scirpt.
I didn't found the same issue on my arm64 server.
But I still found a flaky/random issue on Arm64.
There is a problem usually occurs after 10+ cycles.

Whether I have the patch of TLS PR(#1832) will not affect the result

I will continue to check it.

My script:

while true; do

./bazel-bin/runsc/linux_arm64_pure_stripped/runsc --alsologtostderr --network none --debug --TESTONLY-unsafe-nonroot=true --rootless do ls > test.log 2>&1

STRING=$(cat test.log|grep "no such process")
#                echo $STRING
if [ -n "$STRING" ]; then
   echo "gVisor is broken"
   break
fi

if [ -z "$STRING" ]; then
   echo "gVisor is good"
   continue
fi

done

**Comment by lubinszARM** [comment]
It seems that I can see the same issue of "no such process" on x86 ......

**Comment by avagin** [comment]
> It seems that I can see the same issue of "no such process" on x86 ......

This error isn't fatal. It will be fixed by #2074

**Comment by avagin** [comment]
#1832 has been merged.
